### PR TITLE
Fix 11662 release 1 1

### LINKED ISSUE #11662: Some services of istio have not app selector
**Describe the bug**
Service.spec.selector not has the label app in istio-services:

- pilot
- telemtry
- policy ...

**Expected behavior**
They should have the label in the selector how
istio-ingressgateway

**Version**
Istio 1.1.0-snapshot.5 - January 2019 Release

**Installation**
Official one from docs

**Environment**
Fedora

### PR DESCRIPTION
**Pull request by aljesusg**
Fix #11662 

Can you verify this @jmazzitelli @gbaufake ?

This add the label app selector in services of istio-system, there are some services without this selector.

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/ingress/templates/service.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ spec:
   type: {{ .Values.service.type }}
   selector:
     istio: ingress
+    app: {{ template "ingress.name" . }}
```
Comment (rshriram) [inline install/kubernetes/helm/istio/charts/ingress/templates/service.yaml]: you missed the gateways

Comment (aljesusg) [reply install/kubernetes/helm/istio/charts/ingress/templates/service.yaml]: but gateways service has not label app
https://github.com/istio/istio/blob/9789a6617e75b530980de62a61f1f28202896d6d/install/kubernetes/helm/istio/charts/gateways/templates/service.yaml#L13-L19

**Comment by sdake** [comment]
@aljesusg can you submit this to master please so we may consider a backport?  (also there is a merge collsion)

Cheers
-steve

**Comment by aljesusg** [comment]
Yes, I'll do asap

**Comment by aljesusg** [comment]
Close in favor of https://github.com/istio/istio/pull/11663

**Pull request closed without merging**